buildroot-external/configs/rpi4_64_defconfig

```diff
@@ -42,6 +42,7 @@ BR2_PACKAGE_SQUASHFS=y
 BR2_PACKAGE_RPI_FIRMWARE=y
 BR2_PACKAGE_RPI_FIRMWARE_VARIANT_PI4=y
 BR2_PACKAGE_RPI_WIFI_FIRMWARE=y
+BR2_PACKAGE_RPI_EEPROM=y
```

Comment:
rpi4?

Reply:
I think we can this add also to the rpi4 (32bit) defconfig

Reply:
Added to 32-bit. Left as RPI_EEPROM since I'm sure eventually rpi5 will come out and also use this.